src/dso_api/dynamic_api/remote/views.py

```diff
@@ -102,11 +102,7 @@ def validate(self, serializer, raw_data):
         if not serializer.is_valid():
             raise RemoteAPIException(
                 title="Invalid remote data",
-                detail={
-                    "detail": "These schema fields did not validate:",
-                    "x-validation-errors": serializer.errors,
-                    "x-raw-response": raw_data,
-                },
+                detail="Some schema fields did not validate",
```

Comment:
We could log this information for debugging purposes. Although, when there is private info in there, that should also not end up in the logs. So, better like this.

Reply:
Logging this information is bad indeed. But maybe logging only keys from `raw_data` to error log would be enough for debugging?

Reply:
Good point.

Reply:
Cool, why this was merged then?

Reply:
Because this is a hotfix. Next PR will have a more thorough solution.